default.nix

```
@@ -8,12 +8,12 @@ let
   pkgs = import ./support/nix/nixpkgs.nix { inherit system; };
   inherit (pkgs) lib;
 
-  our-ghc = pkgs.labHaskellPackages.ghcWithPackages (ps: with ps; [
+  our-ghc = pkgs.labHaskellPackages.ghcWithPackages (ps: with ps; ([
     shake directory tagsoup
     text containers uri-encode
     process aeson Agda pandoc SHA
     fsnotify
-  ]);
+  ] ++ (if inNixShell then [ haskell-language-server ] else [])));
```

```suggestion
  ] ++ (if interactive then [ haskell-language-server ] else [])));
```

We don't want this in CI

---

Also this is a mass rebuild for me (144 Haskell packages...). I wonder if we could use a version of nixpkgs/GHC with a binary cache available (if not, I guess the 1lab's cachix will have to do).